src/test/java/com/sawyereffect/cucumber/documentation/ConfluenceMarkupFormatTest.java

```diff
@@ -0,0 +1,122 @@
+package com.sawyereffect.cucumber.documentation;
+
+
+import com.sun.javadoc.*;
+import org.junit.Test;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+import static org.mockito.Mockito.*;
+
+public class ConfluenceMarkupFormatTest {
+
+    @Test
+    public void shouldGetAllClassesDocumentation() throws IOException {
```

Comment:
This test is too big isn't it?

I think there are mixed concerns in this method.

I would say it could be interesting to have a class or method that extracts the information that is required and another one that would be responsible for the output format.